akka-http2-support/src/test/scala/akka/http/scaladsl/Http2ServerTest.scala

```diff
@@ -24,19 +28,30 @@ object Http2ServerTest extends App {
     akka.stream.materializer.debug.fuzzing-mode = off
     akka.actor.serialize-creators = off
     akka.actor.serialize-messages = off
-    akka.actor.default-dispatcher.throughput = 1000
+    #akka.actor.default-dispatcher.throughput = 1000
+    akka.actor.default-dispatcher.fork-join-executor.parallelism-max=8
                                                    """)
   implicit val system = ActorSystem("ServerTest", testConf)
+  import system.dispatcher
 
   val settings = ActorMaterializerSettings(system)
     .withFuzzing(false)
     //    .withSyncProcessingLimit(Int.MaxValue)
     .withInputBuffer(128, 128)
   implicit val fm = ActorMaterializer(settings)
 
+  def slowDown[T](millis: Int): T ⇒ Future[T] = { t ⇒
+    akka.pattern.after(millis.millis, system.scheduler)(Future.successful(t))
+  }
+
   val syncHandler: HttpRequest ⇒ HttpResponse = {
-    case HttpRequest(GET, Uri.Path("/"), _, _, _)      ⇒ index
-    case HttpRequest(GET, Uri.Path("/ping"), _, _, _)  ⇒ HttpResponse(entity = "PONG!")
+    case HttpRequest(GET, Uri.Path("/"), _, _, _)           ⇒ index
+    case HttpRequest(GET, Uri.Path("/ping"), _, _, _)       ⇒ HttpResponse(entity = "PONG!")
+    case HttpRequest(GET, Uri.Path("/image-page"), _, _, _) ⇒ imagePage
+    case HttpRequest(GET, Uri(_, _, p, _, _), _, _, _) if p.toString.startsWith("/image1") ⇒
+      HttpResponse(entity = HttpEntity(MediaTypes.`image/jpeg`, FileIO.fromPath(Paths.get("bigimage.jpg"), 100000).mapAsync(1)(slowDown(1))))
```

Comment:
The ones I used were pretty big intentionally, so we could either use smaller ones or generate ones on the fly instead.